### PR TITLE
Ensure test server is always assigned a random ws/http port above 1023 - Closes #1162

### LINKED ISSUE #1162: WS/HTTP Port Lower Limit requires sudo access to start the lisk instance
Config schema for HTTP and WS port are set to `1-65535`. So user can set these two ports from this limit. While on Linux/Unix `1-1023` are privileged ports and require `sudo` access to start listening on these ports. 

During tests we are generating random ports between range of `1-65535`, if generated lower than `1024` get failed due to above mentioned `sudo` access error. That could also happen to user in starting up lisk instance. 

We need to update schema validation limits and tests accordingly to accepts ports between `1024-65535`

### PR DESCRIPTION
**Pull request by robladbrook**
Closes #1162

### CODE REVIEW

**Comment by karmacoma** [comment]
Once again, thanks for your contribution @robladbrook 👍

I have invited you to our @LiskHQ/community team.

**Comment by karmacoma** [comment]
`1-1023` are privileged ports. Therefore I have tweaked the generated range to `1024-65535` to work within the unprivileged range. I have also changed the schema validations accordingly, as mentioned in the related issue #1162.

Requesting @nazarhussain for further review.

File: schema/config.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ module.exports = {
...
 			wsPort: {
 				type: 'integer',
-				minimum: 1,
+				minimum: 1024,
 				maximum: 65535
 			},
 			httpPort: {
 				type: 'integer',
-				minimum: 1,
+				minimum: 1024,
```
Comment (nazarhussain) [inline schema/config.js]: I believe many users in production environment may run the application on `80` or `443`, who are directly serving lisk instance without a HTTP server. The change in this config may cause problem to them. 

 @karmacoma What do you think?

File: test/common/ws/serverMaster.js [hunk 2]
```diff
@@ -73,16 +74,17 @@ WSServerMaster.generatePeerHeaders = function (headers) {
 	}
 
 	var operatingSystems = ['win32', 'win64', 'ubuntu', 'debian', 'centos'];
+	var httpPort = random.number(1024, 65535);
```
Comment (nazarhussain) [inline test/common/ws/serverMaster.js]: @robladbrook Second param for `random.number` is exclusive range. Should we not use `65536` to make sure we can use `65535` as port?

**Comment by robladbrook** [comment]
@nazarhussain agreed - I've increased the port number as requested

@karmacoma I've reverted your change to set min port number to 1024 because of this line: https://github.com/robladbrook/lisk/blob/109fc7875a3df558c4cd20274f923066646625eb/test/common/ws/serverMaster.js#L84

File: schema/config.js (hunk 1 above)
Comment (karmacoma) [reply schema/config.js]: @nazarhussain Agreed 👍

**Comment by nazarhussain** [comment]
@robladbrook Sorry I missed it initially, reverted my commit.

**Comment by nazarhussain** [comment]
@karmacoma As discussed earlier, reverted the config schema changes, as it may cause trouble for users already running lisk instance on privileged ports with sudo users.

**Pull request merged**